src/components/AddressView.tsx

```diff
@@ -8,20 +9,27 @@ type AddressViewProps = {
 const PREFIX_ADDRESS = "0x";
 
 export const AddressView = ({ address, showChars = 6 }: AddressViewProps) => {
+  const { getStyles } = useMediaContext();
   const username = useZoraUsername(address);
 
   const addressFirst = address.slice(0, showChars + PREFIX_ADDRESS.length);
   const addressLast = address.slice(address.length - showChars);
 
   if (username.username?.username) {
-    return <span>{`@${username.username.username}`}</span>;
+    return (
+      <a {...getStyles("addressLink")} href={`https://zora.co/${username.username.username}`} target="_blank">
```

Comment:
nit: add `rel="noreferrer"` for security/seo reasons

Reply:
Added this to all instances of `target="_blank"`